feat(curriculum): add Build a Bookstore Page Workshop to FSD cert (#62723)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-bookstore-page/68eca2a795b333ca5fee30a8.md

---
id: 68eca2a795b333ca5fee30a8
title: Step 10
challengeType: 0
dashedName: step-10
---

# --description--

> Now create a second book card. Add another `div` element with the `class` attribute set to `card` . Notice how you can reuse the same class name for multiple elements to apply consistent styling.

Reviewer: ```suggestion
Now create a second book card. Add another `div` element with the `class` attribute set to `card`. Notice how you can reuse the same class name for multiple elements to apply consistent styling.
```

# --hints--

> You should have two `div` elements with the `class` of the value `card`.

Reviewer: ```suggestion
You should have two `div` elements with a `class` attribute having a value of `card`.
```

```js

> assert.equal(document.querySelectorAll('.card').length, 2);

Reviewer: ```suggestion
assert.lengthOf(document.querySelectorAll('.card'), 2);
```

```

Your second `.card` element should be inside the `.card-container` element.

```js
assert.equal(document.querySelectorAll('.card-container .card').length, 2);
```

> Your second `.card` element should be a sibling of the first `.card` element.

Reviewer: ```suggestion
The two elements with a class of `card` should be siblings.
```
